packages/salesforcedx-vscode-apex-debugger/package.json

```diff
@@ -59,7 +59,9 @@
     "clean": "shx rm -rf node_modules && shx rm -rf out && shx rm -rf coverage && shx rm -rf .nyc_output",
     "test": "npm run test:vscode-integration",
     "test:vscode-integration": "node ../../scripts/run-vscode-integration-tests-with-top-level-extensions",
-    "test:vscode-insiders-integration": "cross-env CODE_VERSION=insiders npm run test:vscode-integration"
+    "test:vscode-insiders-integration": "cross-env CODE_VERSION=insiders npm run test:vscode-integration",
+    "test:unit": "jest --coverage",
+    "coverage": "jest --coverage"
```

Comment:
I don't think this coverage script is necessary.  It was in one package b/c of how we did coverage previously, but probably doesn't need to be duplicated across packages.

Reply:
This is also present in utils and utils-vscode, but I removed from this package.